HOW_WE_USE_GITHUB.md

```diff
@@ -238,17 +238,180 @@ please post details to the [Nucleus forums](https://community.anaconda.cloud/).
 
 In order to not have to manually type or copy/paste the above repeatedly, please note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option][docs-saved-reply].
 
+---
 
-### Types of Tickets
+## Types of issues
 
-#### Standard Ticket/Issue
+The following section describes the different types of issues that we create in our GitHub
+projects and how/why they are used.
+
+## Standard Ticket/Issue
 
 TODO
 
-#### Epics
+## Epics
 
 TODO
 
-#### Spikes
+## Spikes
 
-TODO
+In this section, we go over the procedure to follow when creating spikes. Exactly what
+these types of issues are and the steps to take when creating them are all outlined
+below. These guidelines apply for all projects falling underneath the
+[conda](github.com/conda) organization.
+
+### What is a spike?
+
+A spike is a term that is borrowed from extreme programming and agile development.
+They are used when the **outcome of a ticket is unknown or even optional**. For example,
+when first coming across a problem that has not been solved before, a team may choose
+to either research the problem or create a prototype in order to better understand it.
+
+Additionally, spikes represent work that **may or may not actually be completed
+or implemented**. An example of this are prototypes created to explore possible solutions.
+Not all prototypes are implemented and the purpose of creating a prototype
+is often to explore the problem space more. For research oriented tasks, the end result
+of this research may be that a feature request simply is not viable at the moment and
+would result in putting a stop to that work.
+
+Finally, spikes are **time boxed**. This means that we set a definite limit on how long
+we want our contributors to work on a spike. We do this to prevent contributors
+from falling into a rabbit hole they may never return from (scary!). Instead, we set
+a time limit, typically between 1 and 3 "days", to perform work on the spike and then
+have the assignee report back to the development team. If the tasks defined in
+the spike have not yet been completed, a decision is made on whether it makes sense
+to perform further work on the spike.
```

Comment:
Given the asynchronous nature of Open Source work, "days" are a bad scale for any contribution formats, including spikes. It suffices to say work on spikes need to be timeboxed and decided case by case to be inclusive to volunteers and non-staff contributors.

```suggestion
from falling into a rabbit hole they may never return from (scary!). Instead, we set
a time limit to perform work on the spike and then have the assignee report back to
the development team. If the tasks defined in the spike have not yet been completed,
a decision is made on whether it makes sense to perform further work on the spike.
```

Reply:
I don't expect contributors to bother with spikes, I see it more as an informative document for anyone who is confused about what we are doing with our planning/what a spike means.

Perhaps we should remove all units from this doc like you're suggesting and include a note admonition that specifies the units we use on our team (e.g., 2-week sprints, "days" for our timeboxes, etc.)?

Reply:
I'm trying to understand why spikes wouldn't be something we could encourage contributors (especially when they are paid to contribute)?

E.g. when they request a feature, they might be able to help with the request by spending time on researching the topic more than the initial issue. I think Jaime and Filipe have both done them, just like many other people in the past.

If we're documenting the Ursa team's choices for "agile development" here, I think this will be seen by potential contributors as the de-facto way to contribute, even if we're not requiring it.

I'd suggest removing the timeframes and just document spikes as "research issues" devoid of any more profound meaning.